sc/src/main/webapp/WEB-INF/jsp/tiles/views/hca-landing-page.jsp

```diff
@@ -0,0 +1,46 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: lingyun
+  Date: 25/10/2018
+  Time: 09:14
+  To change this template use File | Settings | File Templates.
+--%>
```

Comment:
Even though IDEA only wants to help, we don’t include template headers in our code. Authorship and all data can be tracked with Git.